src/components/config/index.ts

```diff
@@ -82,34 +83,58 @@ const handshakeAndValidate = async (config: Config) => {
   }
 }
 
-export const setupConfigFromFile = async (path: string, watch: boolean) => {
```

Comment:
need to revert and simplify this function to be in sync with postman config https://github.com/hyperjumptech/monika/pull/286

Reply:
refactored